### PR TITLE
Adding Android KeywordRecognizer sample.

### PR DESCRIPTION
**Pull request by glecaros**
## Purpose

* Adding Android KeywordRecognizer sample.

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by jhakulin** [comment]
> 	GlobalSection(SolutionConfigurationPlatforms) = preSolution

why UWP things are visible in this PR ?

---
Refers to: quickstart/csharp/uwp/keyword-recognizer/helloworld.sln:9 in 68b7459. [](commit_id = 68b7459cdfb1c3ce8b61ca17b133a0a892dc8ad5, deletion_comment = False)

**Comment by glecaros** [comment]
> > GlobalSection(SolutionConfigurationPlatforms) = preSolution
> 
> why UWP things are visible in this PR ?
> 
> Refers to: quickstart/csharp/uwp/keyword-recognizer/helloworld.sln:9 in [68b7459](https://github.com/Azure-Samples/cognitive-services-speech-sdk/commit/68b7459cdfb1c3ce8b61ca17b133a0a892dc8ad5). [](commit_id = [68b7459](https://github.com/Azure-Samples/cognitive-services-speech-sdk/commit/68b7459cdfb1c3ce8b61ca17b133a0a892dc8ad5), deletion_comment = False)

I removed the BOM to those files (that was breaking the CI process)

File: quickstart/java/android/keyword-recognizer/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+> **Note:**
+> this sample is *not* for the Speech Devices SDK and the Roobo device.
+> If you are looking for further information on these, visit the [Speech Devices SDK](https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-devices-sdk) page.
+
+## Prerequisites
+
+* A PC (Windows, Linux, Mac) capable to run Android Studio.
+* Version 3.1 of [Android Studio](https://developer.android.com/studio/).
+* An ARM-based Android device (API 23: Android 6.0 Marshmallow or higher) [enabled for development](https://developer.android.com/studio/debug/dev-options) with a working microphone.
```
Comment (jhakulin) [inline quickstart/java/android/keyword-recognizer/README.md:14]: >ARM-based [](start = 5, length = 9)

could be also x86 emulator

Comment (glecaros) [reply quickstart/java/android/keyword-recognizer/README.md:14]: We now have x86, ARM and ARM64 so it should work on pretty much every phone/emulator. Will remove the architecture from that statement

File: quickstart/java/android/keyword-recognizer/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Run the sample
+
+* Connect your Android device to your development PC.
+* Press Shift+F10, or select **Run** \> **Run 'app'**.
+* In the deployment target windows that comes up, pick your Android device.
+
+## Note on Android permission handling
+
+Please note that the Speech SDK requires two permissions, i.e., the INTERNET as well as the RECORD_AUDIO permission. As for Android API levels up to 22, requesting the permissions is handled at installation time but from API level 23 users are able to grant/revoke permissions at any time.
```
Comment (jhakulin) [inline quickstart/java/android/keyword-recognizer/README.md:33]: >INTERNET [](start = 68, length = 8)

this is not in the manifest, also not needed for keyword recognizer, right?

Comment (glecaros) [reply quickstart/java/android/keyword-recognizer/README.md:33]: It is not. I kept that as it was talking about the SDK. Rephrased it to talk about the sample

**Comment by jhakulin** [comment]
Thanks for samples! We shall add these to release note

**Comment by BrianMouncer** [comment]
thank you again.

**Pull request merged**